Sources/POSIX/popen.swift

```diff
@@ -82,8 +82,7 @@ public func popen(arguments: [String], redirectStandardError: Bool = false, envi
                 if let str = String.fromCString(buf) {
                     body(str)
                 } else {
-                    //TODO better
-                    fatalError("fatal: popen: couldn't convert buffer to string")
+                    throw SystemError.popen(EILSEQ, arguments[0])
```

Comment:
Needed to throw a `SystemError` here to make everything line up correctly with the `ExitStatus` wrapper. Though, its not technically a system issue. `EILSEQ          Illegal byte sequence` seemed *good enough*. Alternatively, we could add another type to `SystemError`, like `SystemError.encoding` or something.